tests/MongoDB.Bson.Tests/Specifications/uuid/prose-tests/ExplicitDecodingTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using FluentAssertions;
+using MongoDB.Bson.TestHelpers;
+using System;
+using Xunit;
+
+namespace MongoDB.Bson.Tests.Specifications.uuid.prose_tests
+{
+    public class ExplicitDecodingTests
+    {
+        [Fact]
+        public void BsonBinaryData_with_sub_type_3_and_CSharp_byte_order_tests()
+        {
+            using (TemporaryGuidRepresentationMode.V3())
+            {
+                var bytes = BsonUtils.ParseHexString("33221100554477668899aabbccddeeff");
+                var binaryData = new BsonBinaryData(bytes, BsonBinarySubType.UuidLegacy);
```

Comment:
Maybe I should have named this variable `subject`... I was just kind of literally translating from the prose.